src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java

```diff
@@ -121,29 +123,57 @@ public static boolean isPendingInput(WorkflowRun run) {
     }
 
     /**
+     * Deprecated version that coerces {@link GenericStatus} values to the original set (without {@link GenericStatus#QUEUED}).
+     * Most consumers should switch to {@link #computeChunkStatus2(WorkflowRun, MemoryFlowChunk)} and handle unknown status codes.
+     */
+    @CheckForNull
+    @Deprecated
+    public static GenericStatus computeChunkStatus(@Nonnull WorkflowRun run, @Nonnull MemoryFlowChunk chunk) {
+        GenericStatus newStatusEnum = computeChunkStatus2(run, chunk);
+        return (newStatusEnum == GenericStatus.QUEUED) ? GenericStatus.IN_PROGRESS : newStatusEnum;
```

Comment:
So this doesn't address my biggest concern with the shim approach - it's not future-proof. If we add a new `GenericStatus` in the future, we'd have to rename `computeChunkStatus2` to `computeChunkStatus3`, add a new `computeChunkStatus2` that translated the new `GenericStatus`, etc... I know there aren't any plans to add new `GenericStatus`es, but if we're going to go this route, we should really be future-proofing, not a one-off band-aid.